test/easyconfigs/easyconfigs.py

```diff
@@ -616,21 +617,26 @@ def check_python_packages(self, changed_ecs, added_ecs_filenames):
 
         failing_checks = []
 
+        python_default_urls = PythonPackage.extra_options()['source_urls'][0]
+
         for ec in changed_ecs:
 
-            ec_fn = os.path.basename(ec.path)
-            easyblock = ec.get('easyblock')
-            exts_defaultclass = ec.get('exts_defaultclass')
-            exts_default_options = ec.get('exts_default_options', {})
+            with disable_templating(ec):
+                ec_fn = os.path.basename(ec.path)
+                easyblock = ec.get('easyblock')
+                exts_defaultclass = ec.get('exts_defaultclass')
+                exts_default_options = ec.get('exts_default_options', {})
 
-            download_dep_fail = ec.get('download_dep_fail')
-            exts_download_dep_fail = ec.get('exts_download_dep_fail')
-            use_pip = ec.get('use_pip') or exts_default_options.get('use_pip')
+                download_dep_fail = ec.get('download_dep_fail')
+                exts_download_dep_fail = ec.get('exts_download_dep_fail')
+                use_pip = ec.get('use_pip', default=exts_default_options.get('use_pip'))
```

Comment:
I don't think this is correct...

`ec.get('use_pip')` will always work for `PythonBundle`, even if the easyconfig file does not define `use_pip` to `False` or `True`, because `use_pip` is a known easyconfig parameter for `PythonBundle`:

```
$ eb -a -e PythonBundle | grep use_pip | head -1
use_pip*                   Install using 'pip install --prefix=%(prefix)s %(installopts)s %(loc)s' [default: None]
```

So you'll never fall back to `exts_default_options.get('use_pip')`.

The existing line is incorrect too though, we should fall back to `exts_default_options.get('use_pip')` when `use_pip` is still set to `None` (the default) rather than to `False` or `True` explicitly. It's currently wrong when `use_pip = False` is used, but this isn't the correct way of fixing that problem...
```

Reply:
Ah, true. I forgot that those are then set by default